src/Sarif.WorkItems/SarifWorkItemFiler.cs

```diff
@@ -26,15 +49,20 @@ public class SarifWorkItemFiler
         /// current implementation, this context is copied for each SARIF file (if any) split
         /// from the input log and then further elaborated upon.
         /// </param>
-        public SarifWorkItemFiler(SarifWorkItemContext filingContext)
+        public SarifWorkItemFiler(SarifWorkItemContext filingContext) : this(filingContext.HostUri)
         {
             this.FilingContext = filingContext ?? throw new ArgumentNullException(nameof(filingContext));
```

Comment:
>filingContext [](start = 33, length = 13)

It's too late to check for null. We already threw when we dereferenced this to get the argument to pass to the other ctor.